Sources/Datadog/URLSessionAutoInstrumentation/Interception/URLSessionInterceptor.swift

```diff
@@ -20,7 +20,9 @@ public class URLSessionInterceptor: URLSessionInterceptorType {
     }
 
     /// Filters first party `URLs` defined by the user.
-    private let firstPartyURLsFilter: FirstPartyURLsFilter
+    /// **NOTE:** If `session.delegate` is a `DDURLSessionDelegate` initialized with its own
+    /// set of `firstPartyHosts`, then `defaultFirstPartyURLsFilter` is not used
```

Comment:
IMO this comment is unnecessary 💭 - it just explains how the following code works.